mitosheet/src/index.tsx

```diff
@@ -1,11 +1,13 @@
 // Copyright (c) Mito
 
-export * from './version';
+
+
 
 /**
  * By exporting this plugin directly, JLab 4 can get access 
  * directly to the plugin and activate it. Since Notebook 7 is 
  * built on top of JLab 4, this also activates the plugin for Notebook 7.
+ * 
+ * 
+ * Apparently this file is completely unused?? Understand that.
```

Comment:
Investigate this by looking for the main entry point to the extension in tsconfig and setup.py